core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2543,6 +2543,13 @@ public String toString() {
           .setIsHidden(true)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_JOURNAL_BACKUP_WHEN_CORRUPTED =
+      booleanBuilder(Name.MASTER_JOURNAL_BACKUP_WHEN_CORRUPTED)
+          .setDefaultValue(false)
```

Comment:
I think this should be the default behavior? why wouldn't they want a backup when there is an error?